src/Pagination/Filter/AbstractFilterQuery.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Everlution\PaginationBundle\Pagination\Filter;
+
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class AbstractFilterQuery.
+ *
+ * @author Richard Popelis <richard@popelis.sk>
+ */
+abstract class AbstractFilterQuery implements FilterQuery
```

Comment:
Please revert the `Null` objects back as they were. If we are try to avoid duplicate code by introducing Abstract class which does nothing we are doing it wrong ;-)